lib/validate.js

```diff
@@ -822,11 +830,6 @@ function checkSecurity(security,openapi,options) {
 
 function validateSync(openapi, options, callback) {
     setupOptions(options,openapi);
-    if (options.jsonschema) {
```

Comment:
There was some old JSON Schema option which wasnt documented or used, so I killed it.

Reply:
It's documented [here](https://github.com/Mermade/swagger2openapi/blob/master/docs/options.md#options-documentation) and used unless you removed it?

Reply:
Yeah I removed it :) I dont think its a feature we'll need in Speccy.